odxtools/table.py

```diff
@@ -28,6 +29,21 @@ class TableRow:
     key: int
     structure_ref: str
 
+    def __post_init__(self):
+        self._structure: Optional[DopBase] = None
+
+    def _resolve_references(self, id_lookup: Dict[str, Any]) -> None:
+        self._structure = id_lookup.get(self.structure_ref)
+        if self._structure is None:
+            logger.warning(
+                f"STRUCTURE-REF '{self.structure_ref!r}' could not be resolved."
+            )
+
+    @property
+    def structure(self) -> Optional[DopBase]:
+        """The data object property describing this parameter."""
```

Comment:
```suggestion
        """The structure associated with this table."""
```

(looks like a copy-and-pasto to me...)